Makefile

```diff
@@ -107,7 +111,7 @@ obj: $(OBJ)
 
 all: obj app
 
-$(EXE): $(EXE_DEPS)
+$(EXE): $(EXE_DEPS) $(KOBJ)
```

Comment:
Related to other comment it seems necessary to depend on KOBJ explicitly although the object file is in EXE_DEPS...